lib/runtime-core/src/cache.rs

```diff
@@ -207,3 +207,14 @@ pub trait Cache {
     fn load(&self, key: WasmHash) -> Result<Module, Self::LoadError>;
     fn store(&mut self, key: WasmHash, module: Module) -> Result<(), Self::StoreError>;
 }
+
+const WASMER_VERSION: &'static str = env!("CARGO_PKG_VERSION");
+
+/// Returns a unique ID generated from the version of Wasmer for use with cache versioning
+pub fn cache_versioned_sub_directory() -> String {
```

Comment:
I'm not sure if it's really necessary, but do you think it'd be possible to hash the version at compile time? `const` functions aren't advanced enough, but the build.rs would be able to.